mix.exs

```diff
@@ -36,12 +36,14 @@ defmodule Antikythera.Mixfile do
   def project() do
     [
       app:             :antikythera,
-      version:         Antikythera.MixCommon.version_with_last_commit_info("0.1.0"),
+      version:         "0.1.0",
```

Comment:
Versions such as `0.1.0-20180511053513+67ba650c7885eee89ec672d147f1d2c469916f6d` already exist as git commits, although they are not available as hex package.
Should we start with `0.1.1` or `0.2.0`, so that it is strictly newer than all existing versions?
(in this sense, I shouldn't have modified the version from `0.0.1` to `0.1.0` a few weeks ago...)